guides/source/6_1_release_notes.md

```diff
@@ -12,7 +12,7 @@ Highlights in Rails 6.1:
 * Destroy Associations Async
 
 These release notes cover only the major changes. To learn about various bug
-fixes and changes, please refer to the change logs or check out the [list of
+fixes and changes, please refer to the changelogs or check out the [list of
```

Comment:
I'm seeing "change logs" in other release notes as well.
Could you change those as well?

Reply:
Sure. Let me do that.

Reply:
Updated.